npm/webpack-dev-server/test/handlers/angularHandler.spec.ts

```diff
@@ -256,11 +254,11 @@ const expectGeneratesTsConfig = async (devServerConfig: AngularWebpackDevServerC
       outDir: toPosix(path.join(projectRoot, 'out-tsc/cy')),
       allowSyntheticDefaultImports: true,
       skipLibCheck: true,
+      types: ['cypress'],
```

Comment:
So just to confirm, this is the fix?

Reply:
Actually that is not the fix. The issue is because we are using a temp file for the `tsconfig` any types that get added to the extended `tsconfig` `types` array aren't able to resolve. Currently we manually add the Cypress types to the `includes` array. But with this option we don't need to do this and we just just `cypress` to that `types` array.

We could potentially resolve those paths sourcing `typeRoots` from the extended `tsconfig` but it still doesn't allow users to make any additional changes to the tsconfig.  Having a `tsconfig.cypress.json` follows the same ergonomics of a typical Angular project `tsconfig.app.json`, `tsconfig.spec.json`, etc